eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -2459,8 +2459,14 @@
 
   <!-- Machine generated classes for MAA service generate spotBugs errors -->
   <Match>
-    <Class name="~com.azure.security.attestation.models\.[\w]+"/>
-    <Bug pattern="PZLA_PREFER_ZERO_LENGTH_ARRAYS"/>
```

Comment:
Curious what spot bugs errors are you seeing?

Reply:
The generated SDK returns null when a model types returns a null array value instead of an empty array. SpotBugs has a warning that says that empty arrays are preferred over null arrays.

This error is present in the generated code and is propagated to my mirror of the generated code. And I believe it is consistent with the other SDK APIs - when returning an array property which is contained in JSON, if the JSON property isn't present, return null.

The other spotbug failure that several properties were referenced in the .toGenerated method I added that are not initialized in the constructor. The model types in question are temporary-only, I'll be removing them in a subsequent checkin, so I just added an exception. 

My plan is to go back to the exceptions before I'm done with the SDK and ensure that the only ones that are present are the PZLA_PREFER_ZERO_LENGTH_ARRAYS (unless API review says that empty arrays are preferred over null returns for array valued properties).